src/pages/Home/index.js

```diff
@@ -1,13 +1,19 @@
-import React from 'react';
-import './styles.css';
+import React from 'react'
 
-const Home = () => {
+import Navbar from '../../components/Navbar'
+import Banner from '../../components/Banner'
+import CategoryListContainer from '../../components/CategoryListContainer'
+
+import './styles.scss'
 
-    return (
-        <>
-            
-        </>
-    )
-};
+const Home = () => {
+  return (
+    <div>
+      <Navbar />
+      <Banner />
+      <CategoryListContainer />
+    </div>
```

Comment:
You can replace the `div` with nothing

```
<>
  <Navbar />
  <Banner />
  <CategoryListContainer />
</>
```

This is called a `React.Fragment`, it does nothing but to group components (it doesn't appear in the DOM, contrary to the div).. You have to check though, the `div` has defined properties (such as display: block) that may be necessary for your layout.